modules/cert-manager-enable-operator-log-level.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-log-levels.adoc
+
+:_content-type: PROCEDURE
+[id="cert-manager-enable-operator-log-level_{context}"]
+= Setting a log level for the {cert-manager-operator}
+
+You can set a log level for the {cert-manager-operator} to determine the amount of detail in logged error messages. The valid values for the {cert-manager-operator} log level are `Normal`, `Debug`, `Trace`, and `TraceAll`.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* The {cert-manager-operator} is installed.
+
+.Procedure
+
+. Update the deployment for the {cert-manager-operator} to use the config map by running the following command:
```

Comment:
Would prefer removing `config map` as we're just patching the subscription object and making no changes to any config map at this step.

Reply:
How about something like: `Update the subscription object specific to {cert-manager-operator} in order to provide the verbosity level for the operator logs:`